app/src/main/res/values/strings.xml

```diff
@@ -132,11 +132,16 @@
 
     <string name="dashboard_risky_encounter_title">Za posledních 14 dní žádné rizikové setkání</string>
     <string name="dashboard_risky_encounter_subtitle">Poslední aktualizace 14. 5. 2020 v 17:30.\nAktualizace probíhá jednou za 24 hodin</string>
+    <plurals name="dashboard_risky_encounter_title_bad">
```

Comment:
Everything in this file should be in English, not in Czech.

Reply:
Yes

Reply:
up to Tomas, I just replaced a string with plurals